client/src/entrypoints/admin/bulk-actions.js

```diff
@@ -192,6 +230,7 @@ function addBulkActionListeners() {
   document.querySelectorAll(BULK_ACTION_PAGE_CHECKBOX_INPUT).forEach((el) => {
     checkedState.numObjects++;
     el.addEventListener('change', onSelectIndividualCheckbox);
```

Comment:
Just curious - why did you not hook into the existing change event listener?

Reply:
I added separate event for this because click event comes with an inbuilt method called `shiftKey` which returns true if shift-key is pressed. The "change" event lacks this functionality.

Reply:
Nice thanks!